src/main/java/org/opentripplanner/routing/algorithm/mapping/GraphPathToItineraryMapper.java

```diff
@@ -176,9 +176,8 @@ private static CoordinateArrayListSequence makeCoordinates(Edge[] edges) {
 
     /**
      * Slice a {@link State} array at the leg boundaries. Leg switches occur when:
-     * 1. A LEG_SWITCH mode (which itself isn't part of any leg) is seen
-     * 2. The mode changes otherwise, for instance from BICYCLE to WALK
-     * 3. A PatternInterlineDwell edge (i.e. interlining) is seen
+     * 1. The mode changes otherwise, for instance from BICYCLE to WALK
+     * 2. A PatternInterlineDwell edge (i.e. interlining) is seen
```

Comment:
`PatternInterlineDwell` edges should not exist anymore

Reply:
You're right. I've changed the javadoc accordingly.